src/megachatapi_impl.cpp

```diff
@@ -1068,7 +1068,8 @@ void MegaChatApiImpl::sendPendingRequests()
 
                 MegaChatCallPrivate *chatCall = handler->getMegaChatCall();
                 assert(chatCall);
-                if (chatCall->adjustAvFlagsToRestriction(avFlags))
+                EndpointId endPoint(chatroom->chat().client().myHandle(), chatroom->chat().connection().clientId());
+                if (chatCall->adjustAvFlagsToRestriction(avFlags, endPoint))
```

Comment:
The logic in `adjustAvFlagsToRestriction()` needs to be reviewed, @alber2510 
It's not doing what you expect and simply updating the video-flag if *any* flag is changed is not the expected behavior. Here and in the 3 other usages of that method.